src/vision/ds/Image.hx

```diff
@@ -348,7 +348,7 @@ abstract Image(ByteArray) {
 
 		@param x The x coordinate of the pixel.
 		@param y The y coordinate of the pixel.
-		@param color The color to set the pixel to. pay attention to the alpha value.
+		@param color The color to set the pixel to pay attention to the alpha value.
```

Comment:
The period is needed here, you don't set the the pixel because you need to pay attention to the alpha, you just need to set the pixel, and also pay attention to it's alpha value

Reply:
> The period is needed here, you don't set the the pixel because you need to pay attention to the alpha, you just need to set the pixel, and also pay attention to it's alpha value

Oh, ok, let me correct that